packages/devtools-launchpad/src/components/Sidebar.js

```diff
@@ -19,6 +21,35 @@ const Sidebar = React.createClass({
     onSideBarItemClick: React.PropTypes.func.isRequired
   },
 
+  renderTitle(title) {
+
+    return dom.div({className: "title-wrapper"},
+                    dom.h1({}, "Debugger"),
```

Comment:
lets use `title` here...

That'll mean that we get Launchpad/Launchpad... but we could do something cute in that case too... like change the development.json to be "Launchy"

Reply:
How about setting the title property in the development.json to be "Debugger" and adding a new property called "subtitle" with "Launchpad" value?

Reply:
Hmm, I don't anticipate anything other than launchpad at the moment. We might want something in the future, but we can always do that then :)